include/date/date.h

```diff
@@ -1564,7 +1564,7 @@ inline
 month
 operator+(const month& x, const months& y) NOEXCEPT
 {
-    auto const mu = static_cast<long long>(static_cast<unsigned>(x)) + (y.count() - 1);
+    auto const mu = static_cast<long long>(static_cast<unsigned>(x)) + ((int64_t)y.count() - 1);
```

Comment:
Good find.

How about just removing the parentheses around `y.count() - 1`?  Then the compiler will implicitly do the cast.  I'm not sure why I put those parentheses in there in the first place.

Reply:
Thank you. Sounds good about removing the parenthesis.